### PR TITLE
Add a new menu on the website

### PR DESCRIPTION
**Pull request by t-chatoyan**

### CODE REVIEW

**Comment by MartinRst** [comment]
Change the description of Declarative Orchestration it should be: 

Infrastructure as Code for All Your Workflows

Delete the entry for Banking and Healthcare the pages are not ready yet.

Cloud Edition Link should be: https://kestra.io/cloud

otherwise last comments from Nico on Slack and we good.

File: components/layout/Header.vue [hunk 1]
```diff
@@ -96,82 +90,141 @@
...
+<!--                            <li>-->
+<!--                                <NuxtLink class="dropdown-item" href="/use-cases/terraform-provider"-->
+<!--                                    @click="globalClick(true)">-->
+<!--                                    <Terraform />-->
+<!--                                    <p>-->
+<!--                                        <span>Kestra's Terraform Provider</span><br />-->
+<!--                                        Deploy and manage all Kestra resources with Terraform-->
+<!--                                    </p>-->
+<!--                                </NuxtLink>-->
+<!--                            </li>-->
```
Comment (Skraye) [inline components/layout/Header.vue:227]: Remove

File: components/layout/Header.vue [hunk 2]
```diff
@@ -449,47 +539,103 @@ export default {
...
+            left: 0;
+            width: 100%;
             height: 100%;
-            width: 180px;
+            z-index: -1;
+            @supports (-webkit-backdrop-filter: none) or (backdrop-filter: none) {
+                & {
+                    -webkit-backdrop-filter: blur(calc($spacer * 0.625));
+                    backdrop-filter: blur(calc($spacer * 0.625));
+                    background-color: rgb(17 17 19 / 65%);
```
Comment (Skraye) [inline components/layout/Header.vue:590]: Put a variable instead of magic number

File: components/layout/Header.vue [hunk 3]
```diff
@@ -549,32 +708,76 @@ export default {
...
+                    --bs-dropdown-link-active-bg: $black-4;
+                    padding: $spacer;
+                    border-radius: $spacer;
+                    border: 1px solid $black-6;
+
+                    @supports (-webkit-backdrop-filter: none) or (backdrop-filter: none) {
+                        & {
+                            -webkit-backdrop-filter: blur(calc($spacer * 0.625));
+                            backdrop-filter: blur(calc($spacer * 0.625));
+                            background-color: #161617f0;
```
Comment (Skraye) [inline components/layout/Header.vue:745]: put variable instead of magic number

File: components/layout/Header.vue [hunk 4]
```diff
@@ -549,32 +708,76 @@ export default {
...
+                        }
+                    }
+
+                    @include media-breakpoint-down(lg) {
+                        background: #161617E5;
```
Comment (Skraye) [inline components/layout/Header.vue:750]: put variable instead of magic number

File: components/layout/Header.vue [hunk 5]
```diff
@@ -636,6 +840,142 @@ export default {
...
+                            margin-bottom: calc($spacer * 2);
+                        }
+
+                        .dropdown-item {
+                            white-space: unset;
+                        }
+
+                        .column-caption {
+                            font-size: $font-size-base;
+                            font-weight: 600;
```
Comment (Skraye) [inline components/layout/Header.vue:946]: We have no variable for differents font weight ?

**Review by Skraye — CHANGES_REQUESTED** [review]

File: components/layout/Header.vue (hunk 5 above)
Comment (t-chatoyan) [reply components/layout/Header.vue:946]: no we are not using variable for font-weight

File: components/layout/Header.vue (hunk 1 above)
Comment (t-chatoyan) [reply components/layout/Header.vue]: fixed

File: components/layout/Header.vue (hunk 2 above)
Comment (t-chatoyan) [reply components/layout/Header.vue]: fixed

File: components/layout/Header.vue (hunk 3 above)
Comment (t-chatoyan) [reply components/layout/Header.vue]: fixed

File: components/layout/Header.vue (hunk 4 above)
Comment (t-chatoyan) [reply components/layout/Header.vue]: fixed

File: assets/styles/_variable.scss [hunk 6]
```diff
@@ -171,7 +171,9 @@ $box-shadow-lg: 0 1.5rem 1.5rem rgba(#DCE0F9, .5);
 
 // nav
 $nav-pills-border-radius: 50rem;
-
+$menu-dropdown-bg: #161617E5;
```
Comment (Skraye) [inline assets/styles/_variable.scss:174]: magic number

**Review by Skraye — CHANGES_REQUESTED** [review]

**Pull request merged**